SourceX/controls/controller.h

```diff
@@ -2,6 +2,8 @@
 
 #include <SDL.h>
 
+#include "all.h"

```

Comment:
Can we avoid using `all.h` for new code?

Reply:
i don't mind having to clean it up